snap/snapcraft.yaml

```diff
@@ -132,6 +132,14 @@ apps:
       - network
       - network-bind
 
+  opensearch:
```

Comment:
Could it make sense to add a `-bin` suffix? to not confuse users with the `.daemon` ?

Reply:
Not sure. This would be called with `opensearch.opensearch` all the time and the daemon can be generally omitted: `snap start opensearch` knows that it's asking to start the snap app that has `daemon` keyword.

Reply:
I think it is a good switch. We could name it "bin" only, just like we have "cli" @zmraul

Reply:
I'll leave it as `opensearch.opensearch-bin`. Since there is a bit of overlap I would rather be explicit about it.